### PR TITLE
Chore: Add Weekly Docker Prune to On-Prem Servers

### PR DESCRIPTION
**Pull request by krisrjohnson21**
Asana task: [Add periodic Docker prune to Ansible](https://app.asana.com/0/1113545750913664/1207902604792331/f)

This PR adds a weekly `docker prune -a -f` command to the on-prem server crontab. 

This is a new step for me, so I followed [Ansible's builtin cron documentation](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/cron_module.html). I'm unsure of the best way to test this PR, so I'm open to any guidance on that front!

### CODE REVIEW

File: linux/roles/onprem/tasks/prune-docker.yml
Comment (paulswartz) [inline linux/roles/onprem/tasks/prune-docker.yml:1]: suggestion: I'd put this in either the `ecs_anywhere` or `docker` roles; I don't think the `onprem`  role is guaranteed to have Docker installed.

Comment (krisrjohnson21) [reply linux/roles/onprem/tasks/prune-docker.yml:1]: Thanks! Done in https://github.com/mbta/on_prem_deploy/pull/71/commits/0f1547c8f1c2c557407f0c2e9685c65a10b7ef9f

**Review by paulswartz — APPROVED** [review]

**Comment by ianwestcott** [comment]
@krisrjohnson21 @paulswartz FYI these tasks won't be run automatically without a `ansible.builtin.include_tasks` task included in `tasks/main.yml`; see https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_reuse_roles.html#role-directory-structure for more details. (Also that doc references `import_tasks` which is similar but slightly different, TIL: https://serverfault.com/questions/875247/whats-the-difference-between-include-tasks-and-import-tasks)

**Comment by krisrjohnson21** [comment]
> @krisrjohnson21 @paulswartz FYI these tasks won't be run automatically without a `ansible.builtin.include_tasks` task included in `tasks/main.yml`; see https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_reuse_roles.html#role-directory-structure for more details. (Also that doc references `import_tasks` which is similar but slightly different, TIL: https://serverfault.com/questions/875247/whats-the-difference-between-include-tasks-and-import-tasks)

Thank you for flagging this! Addressed in #72

**Pull request merged**